config.go

```diff
@@ -40,7 +40,10 @@ type WAFConfig interface {
 	WithDebugLogger(logger loggers.DebugLogger) WAFConfig
 
 	// WithErrorLogger configures an error logger.
```

Comment:
Should this be renamed to `WithErrorHandler` or something? Guess there isn't any logging-specific thing in this API

Reply:
`WithErrorHandler` gives me the impression I would be able to handle an err (e.g. ignoring it or crashing the entire transaction), something like an exception handler. Maybe `WithErrorCallback`?